src/olympia/scanners/tasks.py

```diff
@@ -49,15 +49,15 @@ def run_scanner(upload_pk, scanner, api_url, api_key):
                                      timeout=settings.SCANNER_TIMEOUT)
 
         try:
-            results = response.json()
+            scanner_results = response.json()
         except ValueError:
             # Log the response body when JSON decoding has failed.
             raise ValueError(response.text)
 
-        if response.status_code != 200 or 'error' in results:
-            raise ValueError(results)
+        if response.status_code != 200 or 'error' in scanner_results:
+            raise ValueError(scanner_results)
 
-        result.results = results
+        result.results = scanner_results
         result.save()
```

Comment:
So we have `result`, `result.results`, `results`, `scanner_results`, and they are all different things... One is a model instance, the others are dicts of data. This is confusing :)

I would do this:
- Rename `scanner_results` to `data`. It's just the response of running the scanner, and we throw it after, so I think we can get away with it having a generic name
- Rename `result` to `scanner_result` (singular), matching the model name (*)
- Keep `results` as it matches the tasks etc.

(*) A bit unfortunate that the model is called `ScannersResult` plural. Maybe we should change that.

What do you think?

Reply:
Yep, that makes sense.

I renamed `result` to `scanners_result` though, to match the model name. I think I used `Scanners` in the model name to match the package name.

Reply:
Renaming the model is a task for another day, but generally the convention in the Django world is to keep the model name singular (matches with the fact that one models are instances of a single object) and the app name plural (logic makes slightly less sense but I like to see the app as hosting multiple instances).

Reply:
oh ok, I see. I'll file an issue to rename the model + package then.

Reply:
Only the model needs to be renamed IMHO - the app name, `scanners`, is already good (and we can consider `yara` as invariable :)

Reply:
oh hu, correct. I misread you. I thought "app" was like "olympia" but that's a project, I guess?

Reply:
Yeah - though addons-server has a custom structure with no real "project" in the Django sense.